pkg/queryfrontend/cache.go

```diff
@@ -34,7 +34,7 @@ func (t thanosCacheKeyGenerator) GenerateCacheKey(userID string, r queryrange.Re
 		i := 0
 		for ; i < len(t.resolutions) && t.resolutions[i] > tr.MaxSourceResolution; i++ {
 		}
-		// Cache key should has a uniq prefix, We use `fe` represent frontend.
+		// Cache key should have a unique prefix. `fe` represents frontend.
```

Comment:
Why do we need to mention `fe` here? Can we remove it?

Reply:
@hanjm let's just remove this comment (:

Reply:
@GiedriusS Thanks. Done.